code/lib/core-server/src/withTelemetry.ts

```diff
@@ -61,7 +61,7 @@ async function getErrorLevel({ cliOptions, presetOptions }: TelemetryOptions): P
 export async function withTelemetry(
   eventType: EventType,
   options: TelemetryOptions,
-  run: () => Promise<void>
+  run: () => Promise<{ port: number }>
```

Comment:
We have to be less precise here, because the run callback is not always `buildDevStandalone`. Let's change this to:

```suggestion
  run: () => Promise<any>
```